canvas_sdk/v1/data/questionnaire.py

```diff
@@ -61,6 +61,27 @@ class Meta:
     show_prologue = models.BooleanField()
     code_system = models.CharField(max_length=255)
     code = models.CharField(max_length=100)
+    enable_behavior = models.CharField(max_length=10, null=True, blank=True)
+
+
+class QuestionEnablementCondition(TimestampedModel):
+    """QuestionEnablementCondition."""
+
+    class Meta:
+        db_table = "canvas_sdk_data_api_questionenablementcondition_001"
+
+    status = models.CharField(max_length=2)
+    question = models.ForeignKey(
+        Question, on_delete=models.DO_NOTHING, related_name="enablement_conditions", null=True
+    )
+    dependent_on = models.ForeignKey(
+        Question, on_delete=models.DO_NOTHING, related_name="dependent_questions", null=True
+    )
+    operator = models.CharField(max_length=10, null=True, blank=True)
+    answer_value = models.CharField(max_length=1000, null=True, blank=True)
+    answer_option = models.ForeignKey(
+        ResponseOption, on_delete=models.DO_NOTHING, related_name="enablement_conditions", null=True
+    )
```

Comment:
These properties are relatively different from the actual defined model (i.e., different `on_delete`, default values, and related names). 
Any reason for this?